contracts/core/extensions/CoreModuleInteraction.sol

```diff
@@ -130,6 +130,34 @@ contract CoreModuleInteraction is
         );
     }
 
+    /**
+     * Converts recipient's components into Set Token's held directly in Vault
+     *
+     * @param _recipient    Address to issue to
+     * @param _set          Address of the Set
+     * @param _quantity     Number of tokens to redeem
+     */
+    function issueInVaultModule(
+        address _recipient,
+        address _set,
+        uint256 _quantity
+    )
+        external
+        nonReentrant
+    {
+        // Require that only modules can call function
+        require(
```

Comment:
Wonder if it helps to have a modifier called `onlyModule`

Reply:
We've generally gone the pattern of eschewing modifiers since they tend to bloat the contract size.